Cooper/ClientApp/src.v2/app/services/language-storage/language-storage.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LanguageStorageService {
+  getLanguage(): string {
+    if (localStorage) {
+        return localStorage.getItem('language') || 'en';
```

Comment:
en - should be constant like DEFAULT_LANGUAGE and also be accessable for other places like property of service